wiki/content/mutations/index.md

```diff
@@ -923,40 +920,24 @@ Result:
       "uid(v)": "0x2"
     }
   },
-  "extensions": {
-    "server_latency": {
-      "parsing_ns": 71033,
-      "processing_ns": 22994018
-    },
-    "txn": {
-      "start_ts": 4,
-      "commit_ts": 5,
-      "preds": [
-        "1-email",
-        "1-name"
-      ]
-    }
-  }
+  "extensions": {...}
 }
 ```
 
-The upsert first checks whether a user with the email `user@dgraph.io` exists. Because
-the database is currently empty, no such user exists. Hence, the variable `v` will be
-empty. In this case, the `uid` function returns a new UID for the variable `v` replacing
-with the new UID value wherever `uid(v)` is used.
-
-### Update Use Case
+The query part of the upsert block stores the UID of the user with the provided email
+id in the variable `v`. The mutation part then extracts the UID from variable `v` and
+stores the `name` and `email` information in the database. In case the user exists,
+the information is updated whereas if the user doesn't exist, `uid(v)` is treated
```

Comment:
If the user exists, its information is updated. If the user doesn't exist, `uid(v)` is treated ...